.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and Publish
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
+jobs:
+  npm-publish:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          # NOTE: Hard Coded Node Version
+          node-version: '18.19.1'
+          registry-url: 'https://registry.npmjs.org'
+      - run: yarn setup
+      - run: ./scripts/publish.sh
```

Comment:
Lets discuss whether we really need a separate publish.sh or should we do a more standard workflow like:

https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

Reply:
This will work. I just need to pass in the new version or `yarn publish` will try to bump it.